lighty-examples/lighty-gnmi-community-restconf-app/example_config.json

```diff
@@ -19,34 +19,23 @@
         },
         "schemaServiceConfig":{
             "topLevelModels":[
+                {" nameSpace": "urn:TBD:params:xml:ns:yang:network-topology","name":"network-topology","revision":"2013-10-21"},
+                {" nameSpace": "urn:opendaylight:params:xml:ns:yang:controller:md:sal:remote","name":"sal-remote","revision":"2014-01-14"},
+                {" nameSpace": "urn:ietf:params:xml:ns:yang:ietf-inet-types","name":"ietf-inet-types","revision":"2013-07-15"},
+                {" nameSpace": "urn:ietf:params:xml:ns:yang:ietf-interfaces","name":"ietf-interfaces","revision":"2014-05-08"},
+                {" nameSpace": "urn:opendaylight:yang:extension:yang-ext","name":"yang-ext","revision":"2013-07-09"},
```

Comment:
Please remove extra space from " nameSpace" -> "nameSpace".  Application started with this configuration will fail.